WordPress/src/main/res/values/strings.xml

```diff
@@ -2254,7 +2254,7 @@
     <string name="site_creation_label_site_details_title">Step 3 of 4</string>
     <string name="site_creation_label_site_details">Tell us more about what you\'re creating. What\'s the title and tagline?</string>
     <string name="site_creation_label_domain_selection_title">Step 4 of 4</string>
-    <string name="site_creation_label_domain_selection">Pick an available \"yourname.wordpress.com\" address to let people find you on the web.</string>
+    <string name="site_creation_label_domain_selection">Pick an available address to let people find you on the web.</string>
```

Comment:
👋 @kwonye , back in the day, we added the "yourname.wordpress.com" in the label as a hint to the user, to make it a bit more clear that the domain suggestion will be for a subdomain to `wordpress.com`. Removing this part might lead to confusion or false expectations, perhaps having people expecting to find suggestions of top-level domain names. Can we re-instate it or maybe expand it to include the `.blog` subdomains?